.github/workflows/publish-release.yml

```diff
@@ -12,14 +12,26 @@ on:
       PUBLISH_DOCS_TOKEN:
         required: true
 jobs:
+  is-release:
+    # Filtering by `push` events ensures that we only release from the `main` branch, which is a
+    # requirement for our npm publishing environment.
+    # The commit author should always be 'github-actions' for releases created by the
+    # 'create-release-pr' workflow, so we filter by that as well to prevent accidentally
+    # triggering a release.
+    if: github.event_name == 'push' && startsWith(github.event.head_commit.author.name, 'github-actions')
+    outputs:
+      IS_RELEASE: ${{ steps.is-release.outputs.IS_RELEASE }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: MetaMask/action-is-release@v1
+        id: is-release
   publish-release:
     permissions:
       contents: write
     # The second argument to startsWith() must match the release-branch-prefix
```

Comment:
Nit: This comment is no longer valid, it was referring to the filter that was removed here